ase/api/src/enum.c

```diff
@@ -194,13 +196,16 @@ fpgaEnumerate(const fpga_properties *filters, uint32_t num_filters,
 		ase_memcpy(&aseToken[1].accelerator_id, readback_afuid, sizeof(fpga_guid));
 	}
 
+	for (i=0; i<2; i++)
+		ase_token[i] = &aseToken[i];
```

Comment:
What's the point of this separate variable?

Reply:
ase_token's type is fpga_token which is the opaque type of void*. 
aseToken's type is the internal data type of struct _fpag_token. In matches_filters, the parameter passed in is "fpga_token *", so the type is void**. In previous code at line 105, the code should dereference the pointer and I found this bug during the negative unit tests for match_filter function. When I fix this bug, I need to change the code here.